core/perf_test/test_atomic_minmax_simple.cpp

```diff
@@ -0,0 +1,221 @@
+// export OMP_PROC_BIND=spread ; export OMP_PLACES=threads
+// c++  -O2 -g -DNDEBUG  -fopenmp
+// ../core/perf_test/test_atomic_minmax_simple.cpp -I../core/src/ -I. -o
+// test_atomic_minmax_simple.x  containers/src/libkokkoscontainers.a
+// core/src/libkokkoscore.a -ldl && OMP_NUM_THREADS=1
+// ./test_atomic_minmax_simple.x 10000000
+
+#include <cstdio>
+#include <cstdlib>
+
+#include <iostream>
+#include <typeinfo>
+
+#include <Kokkos_Core.hpp>
+#include <impl/Kokkos_Timer.hpp>
+
+using exec_space = Kokkos::DefaultExecutionSpace;
+
+template <typename T>
+void sanity(void) {
+  T a = 0;
+  T b = 1;
+  T c;
+
+  std::cout << "atomic_fetch_min:\n";
+  c = Kokkos::atomic_fetch_min(&a, b);
+  std::cout << "a=" << a << "\n";
+  std::cout << "b=" << b << "\n";
+  std::cout << "c=" << c << "\n";
+
+  std::cout << "atomic_fetch_max:\n";
+  c = Kokkos::atomic_fetch_max(&a, b);
+  std::cout << "a=" << a << "\n";
+  std::cout << "b=" << b << "\n";
+  std::cout << "c=" << c << "\n";
+
+  std::cout << "atomic_fetch_max:\n";
+  c = Kokkos::atomic_fetch_max(&a, b);
+  std::cout << "a=" << a << "\n";
+  std::cout << "b=" << b << "\n";
+  std::cout << "c=" << c << "\n";
```

Comment:
Do you want to assert the expected result as well?

Reply:
I should just delete that code.  It was there to debug an error that is now resolved.